itests/util/src/main/java/org/apache/hadoop/hive/cli/control/CoreBeeLineDriver.java

```diff
@@ -97,6 +101,8 @@ public CoreBeeLineDriver(AbstractCliConfig testCliConfig) {
       initScript = new File(testScriptDirectory, testCliConfig.getInitScript());
     }
     cleanupScript = new File(testScriptDirectory, testCliConfig.getCleanupScript());
+    this.replaceHandler = new QTestReplaceHandler();
```

Comment:
Could we use QTestUtil as in other drivers?

Reply:
As I understand form the code, QTestUtil can't be used as is because it doesn't execute commands using beeline.
After digging into other drivers that use QTestUtil, I saw that QTestUtil executes commands from q-files using ReExecDriver.java which lives in hive-ql module, while CoreBeeLineDriver executes commands from q-files using Beeline.java.

Reply:
👍